### PR TITLE
Archiver test doc

### PR DESCRIPTION
**Pull request by syntrust**
A document to reproduce a test to retrieve expired blobs from EthStorage and verify the correctness. (Milestone 5)

### CODE REVIEW

File: ethstorage/archiver/guide-devnet.md [hunk 1]
```diff
@@ -0,0 +1,366 @@
...
+- Container ops-bedrock-l2-1
+- Container ops-bedrock-op-batcher-1
+- Container ops-bedrock-op-node-1
+- Container ops-bedrock-op-proposer-1
+- Container ops-bedrock-op-challenger-1
+- Container ops-bedrock-artifact-server-1
+
+Now, navigate to the parent directory in preparation for the next steps.
+
+The following steps will add an additional OP node in validator mode, configured to sync expired blob data from the es-node. This aims to verify that the functions of the derivation pipeline are working correctly with the BatchInbox contract and the EthStorage archive service.
```
Comment (qzhodl) [inline ethstorage/archiver/guide-devnet.md:268]: Do we need to wait at least 100 slots for some blobs to expire?

File: ethstorage/archiver/guide-devnet.md [hunk 2]
```diff
@@ -0,0 +1,366 @@
...
+--l1.beacon-based-time $GENESIS_TIME \
+--l1.beacon-based-slot 0 \
+--p2p.listen.udp 30375 \
+--p2p.listen.tcp 9733 \
+--rpc.port 9745 \
+--archiver.port 6678 \
+--archiver.enabled 
+```
+
+Now, navigate to the parent directory in preparation for the next steps.
```
Comment (blockchaindevsh) [inline ethstorage/archiver/guide-devnet.md:247]: I suggest to add some comment for this command: how it works / what exactly does it store , though I guess it stores all blobs of blob tx targeted to `$ES_CONTRACT`.

File: ethstorage/archiver/guide-devnet.md (hunk 1 above)
Comment (blockchaindevsh) [reply ethstorage/archiver/guide-devnet.md:268]: Here `OP node` is ambiguous, `OP Stack instance` may be better.

Comment (syntrust) [reply ethstorage/archiver/guide-devnet.md]: The archiver service itself is unaware of dApps. Isn't it?

File: ethstorage/archiver/guide-devnet.md (hunk 2 above)
Comment (syntrust) [reply ethstorage/archiver/guide-devnet.md:251]: The archiver service itself is unaware of dApps. Isn't it?

**Review by qzhodl — APPROVED** [review]

File: ethstorage/archiver/guide-devnet.md (hunk 2 above)
Comment (blockchaindevsh) [reply ethstorage/archiver/guide-devnet.md:251]: I mean for the user of archiver, at first glance he may think an archiver can be shared by multiple dapps.

**Review by blockchaindevsh — APPROVED** [review]

**Pull request merged**